tests/codec/codec_test.go

```diff
@@ -164,7 +162,7 @@ func BenchmarkCodecExample(b *testing.B) {
 		}
 	})
 	// Note: We don't know if wasmer.Wat2Wasm encodes the custom name section or not.
-	// Note: wasmer.Wat2Wasm calls wasmer via CGO which is eventually implemented by wasm-tools
+	// Note: wasmer.Wat2Wasm calls wasmer via CGO which is eventually implemented by internalwasm-tools
```

Comment:
maybe due to replace
```suggestion
	// Note: wasmer.Wat2Wasm calls wasmer via CGO which is eventually implemented by wasm-tools
```